core-services/prow/02_config/_labels.yaml

```diff
@@ -587,6 +587,12 @@ repos:
         target: both
         prowPlugin: label
         addedBy: anyone
+      - color: 5df542
```

Comment:
I think it would be better to change this in a different PR. And after the `label-sync` job succeeds, you can add the label to tide's config.